src/Stratis.Bitcoin.Features.Wallet/Interfaces/IWalletManager.cs

```diff
@@ -102,7 +102,21 @@ public interface IWalletManager
         /// <returns>An unused address or a newly created address, in Base58 format.</returns>
         HdAddress GetUnusedAddress(WalletAccountReference accountReference);
 
-        IEnumerable<HdAddress> GetUnusedAddresses(WalletAccountReference accountReference, int count);
+        /// <summary>
+        /// Gets the first change address that contains no transaction.
```

Comment:
If no addresses are found this will also create new derived addresses in the wallet.